content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,55 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of the build requirements are missing from `dvc.yaml`:
+
+- Files or directories in the <abbr>workspace</abbr> that are dependencies of
+  the stage are missing from `deps` field.
+- Output files or directories that the stage creates, which are already in the
+  workspace, are missing from `outs` field.
+
+Follow the steps below to add existing files/directories as
+<abbr>dependencies</abbr> or <abbr>outputs</abbr> to a stage without
+re-executing it again, which can be expensive/time-consuming, and is
+unnecessary.
+
+We start with an example `prepare`, which has a single dependency and output. To
+add a missing dependency `data.csv`, and output `data/validate` to this stage,
```

Comment:
I'm starting to notice your use of commas is sometimes unnecessary. Not a big deal and sometimes stylistic, but please keep an eye on that @imhardikj. Thanks:
```suggestion
add a missing dependency `data.csv` and output `data/validate` to this stage,
```